charts/ai-software-templates/chatbot/templates/application-gitops-job.yaml

```diff
@@ -8,7 +8,6 @@ metadata:
     app.kubernetes.io/instance: {{ .Release.Name }}-github-repo-creator
     app.kubernetes.io/name:  {{ .Release.Name }}-github-repo-creator
     app.kubernetes.io/part-of: {{ .Release.Name }}
-  namespace: {{ .Release.Namespace }} 
```

Comment:
Same question as the one I registered for the removal of the namespace setting in the app-config.yaml configmap

Reply:
Same as https://github.com/redhat-ai-dev/ai-lab-helm-charts/pull/31#discussion_r1923592267

Reply:
I'm resolving this one, let's continue the discussion on the other comment.